backend/oasst_backend/api/v1/users.py

```diff
@@ -70,6 +72,71 @@ def get_users_ordered_by_display_name(
     return [u.to_protocol_frontend_user() for u in users]
 
 
+@router.get("/cursor", response_model=protocol.FrontEndUserPage)
+def get_users_cursor(
+    lt: Optional[str] = None,
+    gt: Optional[str] = None,
+    sort_key: Optional[str] = Query("username", max_length=32),
+    max_count: Optional[int] = Query(100, gt=0, le=10000),
+    api_client_id: Optional[UUID] = None,
+    search_text: Optional[str] = None,
+    auth_method: Optional[str] = None,
+    api_client: ApiClient = Depends(deps.get_api_client),
+    db: Session = Depends(deps.get_db),
+):
+    def split_cursor(x: str | None) -> tuple[str, UUID]:
```

Comment:
duplicate code?

Reply:
one splits into str, uuid, the other in datetime, uuid ... datetime parsing is the part where the two functions differ .. I can give them different names to highlight this.